public/locales/en/secrets.json

```diff
@@ -1,13 +1,13 @@
 {
   "create": {
-    "title": "Send your secret, securely.",
-    "subtitle": "One time link to share passwords, keys, and other sensitive data.",
+    "title": "Share secrets without the digital footprint.",
+    "subtitle": "Send passwords, keys, and other sensitive data with a single-use link.",
     "secretExpiresAfter": "Secret expires after",
     "button": "Create Link"
   },
   "share": {
     "title": "Done! Just share the link below \uD83C\uDF89",
-    "subtitle": "One time link to share passwords, keys, and other sensitive data.",
+    "subtitle": "Send passwords, keys, and other sensitive data with a single-use link.",
     "thisLinkWillOnlyBeValid": "This link will only be valid for <1>ttl</1>. Once this link is opened or it expires, your secret data will be destroyed from the system.",
     "secureAnotherSecret": "Secure another secret"
   },
```

Comment:
can you change the subtitle below to 'Click \"Open Secret\" to view it. Be sure to save it somewhere safe - it can only be viewed once!'